rpm/libngf-qt5.spec

```diff
@@ -65,11 +76,20 @@ sed 's/Nemo.Ngf/org.nemomobile.ngf/' < declarative/qmldir > %{buildroot}%{_libdi
 
 %postun -p /sbin/ldconfig
 
+%post qtfeedback -p /sbin/ldconfig
+
+%postun qtfeedback -p /sbin/ldconfig
```

Comment:
No libraries in /usr/lib so these shouldn't be needed.

Reply:
Removed these.